agate-core/src/main/java/org/obiba/agate/domain/User.java

```diff
@@ -54,6 +56,8 @@
 
   private DateTime lastLogin;
 
+  private String notificationLocale;
```

Comment:
Usage is not limited to notifications so you should call this property: preferredLanguage